System/Posix/Files.hsc

```diff
@@ -253,7 +253,8 @@ createSymbolicLink :: FilePath -> FilePath -> IO ()
 createSymbolicLink file1 file2 =
   withFilePath file1 $ \s1 ->
   withFilePath file2 $ \s2 ->
-  throwErrnoPathIfMinus1_ "createSymbolicLink" file1 (c_symlink s1 s2)
+  throwErrnoIfMinus1_ ("createSymbolicLink "++file1++" to "++file2)
```

Comment:
For a link (symbolic or hard) this message may be confusing. When we say `ln s1 s2`, we are creating s2 which points to s1, is that a `s1 to s2` link or `s2 to s1`?  The [unix manual for symlink](http://man7.org/linux/man-pages/man2/symlink.2.html) uses the terms `linkpath` (s2) and `target` (s1), maybe we can use the same terminology in the error message here. That way it will also be easy to correlate the error message with syscall manual when debugging.